Changelog.md

```diff
@@ -2,6 +2,7 @@
 
 - **Breaking Change** Renamed `ReactiveCocoaMoyaProvider` to `ReactiveSwiftMoyaProvider`.
 - Renamed the `ReactiveCocoa` subspec to `ReactiveSwift`.
+- `PluginType` can now modify requests and responses through `prepareRequest` and `processResponse`
```

Comment:
Shouldn't be, as there is a default implementation for each function ([here](https://github.com/scottrhoyt/Moya/blob/1b7b08c2c1b405bc6099d1004119a36218a50489/Source/Plugin.swift#L24))